DriveKitVehicleUI/VehiclePicker/View/VehiclePickerTextVC.swift

```diff
@@ -35,9 +35,43 @@ class VehiclePickerTextVC: VehiclePickerStepView {
         if let category = self.viewModel.vehicleCategory {
             textImageView.image = category.categoryImage()
             textDescriptionLabel.attributedText = category.categoryDescription().dkAttributedString().font(dkFont: .primary, style: .normalText).color(.mainFontColor).build()
-            textConfirmButton.configure(text: DKCommonLocalizable.validate.text(), style: .full)
-            textContinueButton.configure(text: "dk_vehicle_category_display_brands".dkVehicleLocalized(), style: .empty)
-            textContinueButton.titleLabel?.textAlignment = .center
+            textConfirmButton.configure(
+                attributedText: "%@\n\n%@"
+                    .dkAttributedString()
+                    .font(dkFont: .primary, style: .smallText)
+                    .buildWithArgs(
+                        "dk_vehicle_detail_category_button_title".dkVehicleLocalized()
+                            .dkAttributedString()
+                            .font(dkFont: .primary, style: .headLine1)
+                            .color(DKUIColors.secondaryColor.color)
```

Comment:
Sur Figma le titre vert est en majuscule 
```suggestion
                            .color(DKUIColors.secondaryColor.color)
                            .uppercased()
```

Reply:
corrigé dans 7157cc8
Du coup, j'ai changé mon fusil d'épaule et j'ai remplacé le `configure(attributedText:...` par un `configure(title: String, subtitle: String? = nil,...)` 
Comme ca, comme pour le style `empty`, le uppercased est inclus dans la config `multilineBordered` et ca simplifie au niveau de l'appel. Par contre, le renommage du premier parametre de `text` à `title` a générer pas mal de changement mais c'est que du renommage